apidocs/.htaccess

```diff
@@ -0,0 +1,5 @@
+# Fixes a vulnerability in CentOS: http://stackoverflow.com/questions/20533279/prevent-php-from-parsing-non-php-files-such-as-somefile-php-txt
+<FilesMatch \.php\.txt$>
+    RemoveHandler .php
+    ForceType text/plain
+</FilesMatch>
```

Comment:
I'd match against `\.php\..*` and remove ForceType directive.
That way ANY file behave like they should and don't get parsed by php...

I didn't test it, don't take my word for it.